pdc_client/__init__.py

```diff
@@ -193,9 +193,9 @@ def get_paged(self, res, **kwargs):
         if self.page_size is not None:
             kwargs['page_size'] = self.page_size
 
-        if self.page_size <= 0 and self.page_size is not None:
-            # If page_size <= 0, pagination will be disable.
-            return res(**kwargs)
+            if self.page_size <= 0:
```

Comment:
I not sure this is right, because None<=0 is true. 
As we know, if pdc client command don't add --page-size parameter, the default value of pagination is 20.